src/univariate/continuous/maxwellboltzmann.jl

```diff
@@ -0,0 +1,61 @@
+immutable MaxwellBoltzmann <: ContinuousUnivariateDistribution
+    a::Float64 # Scale factor
+    chisqd::Chisq
+
+    function MaxwellBoltzmann(a::Real)
+        a > zero(a) || error("MaxwellBoltzmann: a must be positive")
+        @compat new(Float64(a), Chisq(3.0))
+    end
+end
+
+@distr_support MaxwellBoltzmann 0.0 Inf
+
+#### Parameters
+
+scale(d::MaxwellBoltzmann) = d.a
+params(d::MaxwellBoltzmann) = (d.a,)
+
+#### Statistics
+
+mean(d::MaxwellBoltzmann) = 1.5957691216057308 * d.a # 2*sqrt(2/π)
```

Comment:
No reason to write out the computed value since these will be computed at compile time.